scalajvm/app/views/index.scala.html

```diff
@@ -24,30 +24,40 @@
 }
 
 @menu = {
-    <li>
-        Сортировка: @renderOrdering(QuoteOrdering.Time, "по дате")
-        | @renderOrdering(QuoteOrdering.Rating, "по рейтингу")
-    </li>
-    <li>
-        Фильтр: @renderFilter(QuoteFilter.None, "нет")
-        | @renderFilter(QuoteFilter.Year, "за год")
-        | @renderFilter(QuoteFilter.Month, "за месяц")
-        | @renderFilter(QuoteFilter.Week, "за неделю")
-        | @renderFilter(QuoteFilter.Day, "за день")
-    </li>
+    <div class="spaced space-around">
+        <span>Сортировка:&nbsp;</span>
+        <div class="spaced space-around">
+            <span> @renderOrdering(QuoteOrdering.Time, "по дате")</span>
+            <span>&nbsp;| @renderOrdering(QuoteOrdering.Rating, "по рейтингу")</span>
+        </div>
+    </div>
+    <div class="spaced space-around">
+        <span>Фильтр:&nbsp;</span>
+        <div class="spaced space-around">
+            <span>  @renderFilter(QuoteFilter.None, "нет")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Year, "за год")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Month, "за месяц")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Week, "за неделю")</span>
+            <span>&nbsp;| @renderFilter(QuoteFilter.Day, "за день")</span>
```

Comment:
It's important to have non-breaking spaces here, so I decided to use an HTML entity instead of actual non-breaking space (“ ”) to make it more apparent.